khi_robot_control/package.xml

```diff
@@ -34,6 +34,11 @@
   <run_depend>trajectory_msgs</run_depend>
   <run_depend>transmission_interface</run_depend>
   <run_depend>khi_robot_msgs</run_depend>
+  <run_depend>khi_duaro_moveit_config</run_depend>
+  <run_depend>khi_rs007n_moveit_config</run_depend>
+  <run_depend>khi_rs007l_moveit_config</run_depend>
+  <run_depend>khi_rs080n_moveit_config</run_depend>
```

Comment:
Just a comment: making MoveIt a dependency of your `control` package seems like a really heavy dependency.

In essence this means that I cannot install just `khi_robot_control` any more without also installing (all of) MoveIt (which will require RViz, OpenGL, a graphical desktop environment, etc). For more embedded / headless deployments of `khi_robot_control`, that seems prohibitively costly.

Is this dependency "only" added to get access to the `joint_limits.yaml` file?

If so: may I suggest to store those limits files in the description packages instead? They can be considered part of the "description" of the robot's kinematics and dynamics, so storing them with the robot models would make sense, and would avoid needing to depend on MoveIt from `khi_robot_control`.

Reply:
@gavanderhoorn Yes, it is just for `joint_limits.yaml`. I agree with that suggestion, so I added a commit "Move joint_limits.yaml from moveit_config to description" to do that.